src/h3/_cy/__init__.py

```diff
@@ -42,12 +47,12 @@
     # compact,
     # uncompact,
     # num_hexagons,
-    mean_hex_area,
-    cell_area,
-    line,
+    # mean_hex_area,
+    # cell_area,
+    # line,
     # is_res_class_iii,
-    get_pentagon_indexes,
-    get_res0_indexes,
+    # get_pentagon_indexes,
+    # get_res0_indexes,
```

Comment:
I would just delete these lines. They are not planned to be uncommented.

Reply:
My plan (which I probably haven't communicated well) was that once we finish the migration to remove `h2fake2` that this file will be unchanged compared to the v3 version on `master`. So eventually, yes, they will be uncommented.

I agree that all these function names should be changed to something better, I just want to avoid that to minimize the surface area of what we're changing at this step.

Reply:
And to be clear, this file will eventually change with better names, just not at this stage.